cognite/neat/_client/_api/schema.py

```diff
@@ -48,3 +50,72 @@ def retrieve(
             views=views,
             containers=ContainerApplyDict(containers.as_write()),
         )
+
+    def retrieve_data_model_id(self, data_model_id: dm.DataModelIdentifier) -> DMSSchema:
+        data_models = self._client.data_modeling.data_models.retrieve(data_model_id, inline_views=True)
+        if len(data_models) == 0:
+            raise NeatValueError(f"Data model {data_model_id} not found")
+        data_model = data_models.latest_version()
```

Comment:
Uh great ! there is method for getting latest version ! I was not aware it exists